packages/composables/package.json

```diff
@@ -60,6 +60,7 @@
     "@shopware-pwa/helpers-next": "workspace:*",
     "@shopware/api-client": "workspace:*",
     "@vueuse/core": "^10.7.1",
+    "defu": "^6.1.3",
```

Comment:
Why do we need a whole package?
There was a file for deep merging translations.

Reply:
Comment from Patryk before:
`remove this helper and use [defu](https://www.npmjs.com/package/defu) instead, well tested, more powerful and small`

Reply:
Yeah, mostly as it was untested code and maintenance burden 👌